src/julia.h

```diff
@@ -2289,6 +2289,16 @@ typedef struct _jl_task_t {
     // uint8_t padding1;
     // multiqueue priority
     uint16_t priority;
+    // flag indicating whether or not this to measure the wall and cpu time of this task
+    uint8_t is_timing_enabled;
+    // timestamp this task first entered the run queue (TODO: int32 of ms instead?)
+    uint64_t first_enqueued_at;
+    // timestamp this task was most recently scheduled to run
+    uint64_t last_dequeued_at;
```

Comment:
small nitpicky suggestion, but i am not in love with the name `last_dequeued_at`. I think it is precise, but not as clear as it could be.

Would `last_scheduled_to_run_at` be too verbose? Or `last_scheduled_on_thread_at`? Or `last_started_running_at`?

Also, when picking these names, maybe we can keep in mind that we might add a few more states in the near future for tracking compilation time, and different wait time reasons, so we might end up with things like:
- `last_started_waiting_at`
- `last_started_compiling_at`

etc, so i think i like `last_started_running_at` the best! :)

Reply:
Happy to change names.

I mostly just want to avoid using the word `scheduled` here, since it can be ambiguous / confusing (since `schedule(t)` means "(re)insert `t` into the scheduler run queue", but when someone asks "when was `t` last scheduled?" they usually mean "when did `t` last start running?" not "when did `t` last get (re)inserted to the run queue?".

`first_enqueued_at` and `last_started_running_at` seem fine to me. 

(These should be internal names anyway, and we should have public APIs like `task_cpu_time()` or whatever that are actually public/stable, but still best if we pick good ones to begin with).

Reply:
Cool, +1

Reply:
done